package.json

```diff
@@ -41,18 +41,38 @@
       ]
     },
     "configuration": {
-      "title": "Search JSON Path configuration",
+      "title": "Search JSON Path",
       "type": "object",
       "properties": {
-        "search-json-path": {
-          "enable": true,
-          "prefixAndSuffix": [
+        "search-json-path.enable": {
+          "type": "boolean",
+          "default": true,
+          "description": "enable addon"
+        },
+        "search-json-path.prefixAndSuffix": {
```

Comment:
![image](https://github.com/guilhermeavanci/vscode-search-json-path/assets/5530960/14d4de0e-54fd-4a69-b037-0cbaa907c57d)
I want the configuration GUI to actually work.  maybe we can do that by changing the type or doing some other configuration?

I'm having difficulty understand why  this is the type it is.  Maybe we should reshape this config to make it clear what is being represented?